ndcube/ndcube.py

```diff
@@ -366,6 +367,48 @@ def extra_coords(self):
                     "value": self._extra_coords_wcs_axis[key]["value"]}
         return result
 
+    @property
+    def all_world_coords(self):
```

Comment:
This should probably take a axes input.

Also it should take in any unique substring of `world_axes_physical_types` or an int.

Reply:
And it should also be a method rather than a property.